File: templates/androidstarters-java/app/src/commonTest/java/io/mvpstarter/sample/common/injection/component/TestComponent.java
package <%= appPackage %>.common.injection.component;

import javax.inject.Singleton;

import dagger.Component;
import <%= appPackage %>.common.injection.module.ApplicationTestModule;
import <%= appPackage %>.injection.component.AppComponent;

@Singleton
@Component(modules = ApplicationTestModule.class)
public interface TestComponent extends AppComponent {
}

File: templates/androidstarters-kotlin/app/src/commonTest/java/io/mvpstarter/sample/common/injection/component/TestComponent.kt
package <%= appPackage %>.common.injection.component

import dagger.Component
import <%= appPackage %>.common.injection.module.ApplicationTestModule
import <%= appPackage %>.injection.component.AppComponent
import javax.inject.Singleton

@Singleton
@Component(modules = arrayOf(ApplicationTestModule::class))
interface TestComponent : AppComponent